packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -722,6 +736,11 @@ const New = decorate([
         state.missingExportRetention ||
         state.missingCopyRetention ||
         state.missingSnapshotRetention,
+      isRetentionLimit: ({
```

Comment:
Find a better name for this computed.

Maybe something like `isRetentionTooHigh`.